src/components/tipRecords/AuthorAndDate.vue

```diff
@@ -67,11 +70,6 @@ export default {
         this.hover = hover;
       }, 500),
     },
-    ...mapState({
-      name({ chainNames }) {
```

Comment:
so, `chainNames` doesn't contain all needed names anymore?

Reply:
guess so

Reply:
they are now all included in the respective backend response, before it was duplicate from state and backend as well

Reply:
it's still needed, should be resolved now